src/setup/BitPaySetup.ts

```diff
@@ -1,5 +1,3 @@
-import fetch from 'node-fetch';
```

Comment:
Since we support a minimum of Node 18.x, we do not need `node-fetch`. It was also removed by `msw`.